packages/contracts/src/dollar/core/CreditNFT.sol

```diff
@@ -55,30 +56,30 @@ contract CreditNFT is ERC1155Ubiquity {
 
         //insert new relevant block number if it doesn't exist in our list
         // (linked list implementation won't insert if dupe)
-        _sortedBlockNumbers.pushBack(expiryBlockNumber);
+        require(_sortedBlockNumbers.pushBack(expiryBlockNumber));
```

Comment:
Was this (lack of `require`) a vulnerability picked up by Slither etc?

Reply:
Yes, both the EnumerableSet and StructurededLinkList will return a bool for functions that alter the sets. Slither was calling these out as being ignored and a require was the simplest way to ensure a revert on failure.